pkg/operator/ceph/object/topic/provisioner.go

```diff
@@ -99,6 +103,7 @@ func createSNSClient(p provisioner, objectStoreName types.NamespacedName) (*sns.
 			WithEndpoint(objContext.Endpoint).
 			WithMaxRetries(3).
 			WithDisableSSL(!tlsEnabled).
+			WithHTTPClient(&client).
```

Comment:
Is this part of the TLS change?

Reply:
@leseb : This was missing in the intial PR, for accessing normal RGW endpoint, the default http client was sufficient hence it was not failing. But if try to access the TLS enabled RGW with default client it will fail